pkgs/development/libraries/libvpx/default.nix

```diff
@@ -135,16 +135,10 @@ stdenv.mkDerivation rec {
                     experimentalFpMbStatsSupport ||
                     experimentalEmulateHardwareSupport) "experimental")
   ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-    #"--extra-cflags="
-    #"--extra-cxxflags="
-    #"--prefix="
-    #"--libc="
-    #"--libdir="
-    "--enable-external-build"
     # libvpx darwin targets include darwin version (ie. ARCH-darwinXX-gcc, XX being the darwin version)
     # See all_platforms: https://github.com/webmproject/libvpx/blob/master/configure
     # Darwin versions: 10.4=8, 10.5=9, 10.6=10, 10.7=11, 10.8=12, 10.9=13, 10.10=14
-    "--force-target=${stdenv.hostPlatform.config}${
+    "--force-target=${stdenv.hostPlatform.parsed.cpu.name}-${stdenv.hostPlatform.parsed.kernel.name}${
```

Comment:
Nice. I strugged with this one. I think so on aarch64 it is still incorrect because `arm64` != `aarch64`.

Reply:
I was able to build for aarch64. I think they rewrite it here:

https://github.com/webmproject/libvpx/blob/0c46742f56d06bf284c781bd754eff3b1d97c3c8/build/make/configure.sh#L723